recipes-utils/exfatprogs/exfatprogs/run-ptest

```diff
@@ -0,0 +1,78 @@
+#!/bin/sh
+
+rm -f ptest.log || true
+rm -f test_create.log || true
+
+loopdev=$(losetup -f)
+
+mountdir="/mnt/exfatprogs_ptest"
+
+if [ ! -d "${mountdir}" ]; then
+  mkdir "${mountdir}"
+fi
+
+exfat_support=$(grep exfat /proc/filesystems >/dev/null ; echo $?)
+module_installed=0
+
+if [ "${exfat_support}" = "1" ]; then
+  insmod /lib/modules/$(uname -r)/kernel/fs/exfat/exfat.ko
+  if [ $? -ne 0 ]; then
+    echo "Error: This kernel is not support exfat filesystem."
```

Comment:
"Error: This kernel is not supporting exfat filesystem." or ""Error: This kernel does not support exfat filesystem.""